postgresql-plugin/src/e2e-test/features/postgresql/source/PostgresqlRunTimeMacro.feature

```diff
@@ -0,0 +1,281 @@
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "PostgreSQL" from the plugins list as: "Sink"
+    Then Connect plugins: "PostgreSQL" and "PostgreSQL2" to establish connection
+    Then Navigate to the properties page of plugin: "PostgreSQL"
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "driverName"
+    Then Replace input plugin property: "host" with value: "host" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "port" for Credentials and Authorization related fields
+    Then Replace input plugin property: "user" with value: "username" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "password" for Credentials and Authorization related fields
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Replace input plugin property: "database" with value: "databaseName"
+    Then Click on the Macro button of Property: "importQuery" and set the value in textarea: "PostgreSQLInvalidImportQuery"
+    Then Validate "PostgreSQL" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "PostgreSQL2"
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "driverName"
+    Then Replace input plugin property: "host" with value: "host" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "port" for Credentials and Authorization related fields
+    Then Replace input plugin property: "database" with value: "databaseName"
+    Then Click on the Macro button of Property: "tableName" and set the value to: "PostgreSQLTableName"
+    Then Click on the Macro button of Property: "dbSchemaName" and set the value to: "PostgreSQLSchemaName"
+    Then Replace input plugin property: "user" with value: "username" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "password" for Credentials and Authorization related fields
+    Then Enter input plugin property: "referenceName" with value: "targetRef"
+    Then Validate "PostgreSQL2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Enter runtime argument value "invalidTableNameImportQuery" for key "PostgreSQLInvalidImportQuery"
+    Then Enter runtime argument value "invalidTable" for key "PostgreSQLTableName"
+    Then Enter runtime argument value "schema" for key "PostgreSQLSchemaName"
+    Then Run the preview of pipeline with runtime arguments
+    Then Verify the preview of pipeline is "Failed"
+
+  @POSTGRESQL_SOURCE_TEST @POSTGRESQL_SINK_TEST @BQ_SINK_TEST
+  Scenario: To verify data is getting transferred from PostgreSQL source to BigQuery sink using macro arguments
```

Comment:
We have wriiten the scenario for connection section only, Basic section is left I believe can we add a test ?

Reply:
Done.